### PR TITLE
Correct documentation errors

### LINKED ISSUE #171: Doc issues need fixing
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Hi, Peter. Just an FYI. I looked at the HTML archive for your PyAdditive PR #190 to see if I could see the differences in the formatting between specifying ``None`` versus ``default: None`` for a parameter. However, I got sidetracked by some other issues/questions. Here are some screen shots showing that API summary tables are missing descriptions for Python objects and that sometimes the brief descriptions are too long to fit in the space allocated and so should be shortened if at all possible. (More descriptive information can always be added in complete sentences after a blank line. This is why descriptions start with a brief fragment.) I don't know why the text size for the descriptions in these summary tables is smaller than normal, but the PyAnsys team probably made it this way to maximize the number of words that can be shown.) Addressing missing descriptions, truncated descriptions, formatting issues, and default values is likely something you will want to do in later PRs. If you want, you can copy all of this into an issue on the repository. I advise using the artifact to page through and look for issues such as these!
 
**Missing description examples**
6ddb961e-8849-4522-a8ce-550bc44dde77

14a00791-bd33-476e-95e2-59ad8870223a
additive/assets/48300564/97440bb4-6a17-4b54-b612-a99c6caada40)

**Truncated description examples**
1aa2c238-3444-4c92-8bf1-54ba19e3f512

9fc8acab-3aa6-4596-a5ba-51cb4a2c8f9d

 
**Formatting issue example**
This formatting in the "Returns" section appears many times:
a10892a6-449f-4f85-856b-0a6cc513e68b

### 📝 Steps to reproduce

tox -e doc

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

_No response_

### 🐍 Which Python version are you using?

3.9

### 📦 Installed packages

```shell
n/a
```

### PR DESCRIPTION
**Pull request by pkrull-ansys**
fixes #171

### CODE REVIEW

File: src/ansys/additive/core/download.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
+"""Provides a function to download files from the server to the client."""
```
Comment (julieatansys) [inline src/ansys/additive/core/download.py:22]:
```suggestion
"""Provides a function for downloading files from the server to the client."""
```

File: src/ansys/additive/core/examples/downloads.py [hunk 2]
```diff
@@ -119,18 +123,18 @@ def download_small_wedge_slm_build_file():
...
+    """Container for the example material tuning process inputs.
 
     Parameters
     ----------
     experiment_data_file : str
-        Path to the experiment data file.
+        Path to the experiment data file (CSV).
     material_parameters_file : str
-        Path to the material parameters file.
+        Path to the material parameters file (JSON).
```
Comment (julieatansys) [inline src/ansys/additive/core/examples/downloads.py:133]:
```suggestion
        Path to the material parameters file (JSON).
```
Don't we call this the material configuration file?

File: src/ansys/additive/core/geometry_file.py [hunk 3]
```diff
@@ -19,7 +19,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
-
+"""Provides a container for files describing a part geometry."""
```
Comment (julieatansys) [inline src/ansys/additive/core/geometry_file.py:22]:
```suggestion
"""Provides a container for part geometry files."""
```

File: src/ansys/additive/core/parametric_study/constants.py [hunk 4]
```diff
@@ -19,6 +19,7 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
+"""Provides constant values related parametric studies."""
```
Comment (julieatansys) [inline src/ansys/additive/core/parametric_study/constants.py:22]:
```suggestion
"""Provides constant values related to parametric studies."""
```

File: src/ansys/additive/core/simulation.py [hunk 5]
```diff
@@ -31,25 +31,25 @@
...
-    """Simulation status values."""
+    """Provides simulation status values."""
 
-    #: Simulation is awaiting execution.
     PENDING = "Pending"
-    #: Simulation was executed.
+    """Simulation is awaiting execution."""
     COMPLETED = "Completed"
-    #: Simulation errored.
+    """Simulation was executed."""
```
Comment (julieatansys) [inline src/ansys/additive/core/simulation.py:48]:
```suggestion
    """Simulation was run."""
```

**Review by julieatansys — APPROVED** [review]

File: src/ansys/additive/core/material.py [hunk 6]
```diff
@@ -701,8 +689,7 @@ def strain_scaling_factor(self, value: float):
 
     @property
     def support_yield_strength_ratio(self) -> float:
-        """Factor to reduce the yield strength and elastic modulus of the
-        support material."""
+        """Factor to reduce the yield strength and elastic modulus of the support material."""
```
[the remark this answers is not in the document]
Comment (pkrull-ansys) [reply src/ansys/additive/core/material.py:692]: @julieatansys FYI, the first line of the doc string (everything within triple quotes) is considered the summary. "First line" is taken literally by the doc builder. Thus, the first line needs to be a complete sentence without any carriage returns in it. A blank line needs to be between the summary and subsequent explanation. If the summary line exceeds 110 characters, a special comment, `# noqa`, needs to be placed after the last triple quote so the line length check doesn't get trigger. 110 characters is a PyAdditive configuration setting. Other libraries may differ.

I'll reconfigure your suggestion and add it.

File: src/ansys/additive/core/examples/downloads.py (hunk 2 above)
Comment (pkrull-ansys) [reply src/ansys/additive/core/examples/downloads.py]: Changed `material parameters file` to `material configuration  file`.

**Pull request merged**